tools/agent-ctl/Cargo.lock

```diff
@@ -985,7 +985,6 @@ dependencies = [
  "serde_json",
  "slog",
  "slog-scope",
- "tempfile",
```

Comment:
As discussed on our call, the Cargo.lock update is probably due to an earlier change forgetting to check in the Cargo.lock update.  I'd therefore split this out as a separate patch.

Reply:
@dgibson I rebased and this disappeared, however other diff appeared, I just left it out of the PR